config/config_test.go

```diff
@@ -113,14 +98,8 @@ func testValidateTesthelper(
 		test.That(t, cfg.Sensors, test.ShouldResemble, cfgService.Attributes["sensors"])
 		test.That(t, *cfg.DataRateMsec, test.ShouldEqual, cfgService.Attributes["data_rate_msec"])
 		test.That(t, cfg.ConfigParams, test.ShouldResemble, cfgService.Attributes["config_params"])
-
-		if cloudStoryEnabled {
-			test.That(t, cfg.ExistingMap, test.ShouldEqual, cfgService.Attributes["existing_map"])
-			test.That(t, cfg.MapRateSec, test.ShouldBeNil)
-		} else {
-			test.That(t, cfg.DataDirectory, test.ShouldEqual, cfgService.Attributes["data_dir"])
-			test.That(t, *cfg.MapRateSec, test.ShouldEqual, cfgService.Attributes["map_rate_sec"])
-		}
```

Comment:
test revert to reflect config validation revert so that we can make config changes without breaking cartographer